app/components/task-details.hbs

```diff
@@ -0,0 +1,12 @@
+<div class="task-details">
+  <div class="task-title"><b>Title: </b>{{@task.title}}</div>
+  <div class="task-purpose"><b>Purpose: </b>{{@task.purpose}}</div>
```

Comment:
If these are single lines, why not put them in a `<p>` instead of a `<div>`
A couple of reasons are: 
- Accessibility 
- Readability 
- Good convention and practices

I'd be happy to chat more if you'd like to have a discussion on the same

Reply:
Same on line 6 and 9, as well

Reply:
Or `<span>` 😄

Reply:
If you'd like them separated by a line, then it'd be alright to go ahead with a `p` tag
Thanks for clarifying 😄

Reply:
done